internal/cmd/iam/command_provider_update.go

```diff
@@ -8,9 +8,10 @@ import (
 	pcmd "github.com/confluentinc/cli/internal/pkg/cmd"
 	"github.com/confluentinc/cli/internal/pkg/errors"
 	"github.com/confluentinc/cli/internal/pkg/examples"
-	"github.com/confluentinc/cli/internal/pkg/output"
 )
 
+const identityProviderNoOpUpdateErrorMsg = "one of `--description` or `--name` must be set"
```

Comment:
Does this get used in multiple places in this package? If not, no need to create a constant (it just makes you have to jump around more when you're reading the code)